src/dswx_sar/dswx_ni_runconfig.py

```diff
@@ -226,6 +252,28 @@ def get_freq_rtc_hdf5(input_rtc):
 
     return freq
 
+def get_res_rtc_hdf5(input_rtc, freq_group):
```

Comment:
@hb6688 Let's change the function name. You are reading `spacing`, not `resolution`. 
```suggestion
def get_x_spacing_rtc_hdf5(input_rtc, freq_group):
```